fuse.spec

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
Is there a reason we don't update to a later version and then apply a minimal set of patches?   Is this the version on rh?   Looks like latest libfuse is 3.16.2

Reply:
I'm considering a patch to statically link the libfuse library only.  This would allow us to set this to the latest for our build without affecting whatever the system wants to use.

Reply:
https://github.com/daos-stack/daos/pull/13749

Obviously, this would require some changes to this repo and some testing of the same.  But I tested it locally and it removes the runtime dependence on libfuse3.so and seems to work.

Reply:
> For context, I'm considering a patch to statically link the libfuse library only. This would allow us to set this to the latest for our build without affecting whatever the system wants to use.

We **definitely** don't want to static link.  Doing so would require that we do an entire release cycle including all of the manual QA and SDL and whatnot in order to fix a bug [security or otherwise] in the libfuse that we statically link with.  With dynamic linking we can release just a fuse update and users can install that to resolve whatever issues there are.  Much less overhead on our part.

Reply:
> Is there a reason we don't update to a later version and then apply a minimal set of patches?

Because our ultimate goal is to be able to continue to use the package from RH once they patch what we want into it.

> Is this the version on rh? Looks like latest libfuse is 3.16.2

Keep in mind that during a release RH prefers to keep their base the same and add patches to address features and bugs so the version marked on a package does not necessarily reflect the features/bugs that have been addressed in it.